src/Mod/Part/App/PrimitiveFeature.cpp

```diff
@@ -652,6 +652,13 @@ App::DocumentObjectExecReturn *Cone::execute()
         return new App::DocumentObjectExecReturn("Radius of cone too small");
     if (Height.getValue() < Precision::Confusion())
         return new App::DocumentObjectExecReturn("Height of cone too small");
+    double angle = Angle.getValue();
+#if OCC_VERSION_HEX == 0x070702
+    // OCCT 7.7.2 will not model a cone with an angle of exactly 360, so we cheat:
+    if ( angle == 360.0) {
+5        angle = 359.99;

```

Comment:
I think the '5' may be a typo.

Reply:
Yep.  I blame having fortran on my brain  ( bad punched card joke ).  Thanks!